script.js

```diff
@@ -98,3 +99,57 @@ Resume = {
 		}
 	]
 };
+app.workCounter = 0;
+
+app.moreWork = function() {
+	// append more work info
+	app.workCounter++;
+	console.log("fizz");
+	const workToAppend = `<fieldset class="work${app.workCounter}" data-jsonSection="work${app.workCounter}">${$(
+		".work0"
+	).html()}</fieldset>`;
```

Comment:
I love the way you give each one a unique number

Reply:
thank you! I got kind of caught up in this hahaha